Notatnik_parser/Notatnik_parser/Notepad/NotepadMenu.cs

```diff
@@ -6,25 +6,23 @@ namespace Notatnik_parser.Notepad
 {
     public class NotepadMenu
     {   
-        string note;
-
-        public void SaveFile()
+        private string SaveFile()
```

Comment:
now, this methos is responsible for two functionalities: get text from console and save it to file. Try create private method for get text and add string parameter to SaveFile function. Then you can use them in Run()